### PR TITLE
Scene Tree: double click on subscene icon will open it

### PR DESCRIPTION
**Pull request by kubecz3k**
It's possible to double click scene icon inside tree view, to open it quickly:
quick_open_subscene

### CODE REVIEW

File: editor/scene_tree_editor.h [hunk 1]
```diff
@@ -141,6 +141,8 @@ class SceneTreeEditor : public Control {
 	List<StringName> *script_types;
 	bool _is_script_type(const StringName &p_type) const;
 
+	int subscene_previously_clicked_at_time = 0;
```
Comment (bojidar-bg) [inline editor/scene_tree_editor.h]: I think this won't compile on old compilers, move initialisation into constructor?

Comment (kubecz3k) [inline editor/scene_tree_editor.h]: oh yeah, remember from previous pr that you are right, thanks!

**Comment by reduz** [comment]
Didn't doubleclick focus on the node in the scene?

**Comment by kubecz3k** [comment]
@reduz double click on the normal icon will focus. Double click on that 'subscene' icon (the one that normally shows popup) does not.

**Comment by reduz** [comment]
oh, i see

**Comment by toger5** [comment]
This is more a question to educate me than a proposal. (just to make sure this does not sound as if I think I know things better)

Is that the normal way of implementing double clicks (`subscene_previously_clicked_at_time`)? Since `InputEventMouseButton` supports `click_count` (or similar) could that be implemented as another button signal to make double click easier?

**Comment by groud** [comment]
Agree with @toger5 , It is pretty hacky to have the double click interval time hardcoded here. This should be a global parameter of the editor.

**Comment by kubecz3k** [comment]
After some consultation it seems there is no straightforward way to implement this in less hacky way. We came to conclusion that single click on this icon should open scene, and all the options should be moved to node right click menu. 
Closing this, since changes that we need in order to implement this functionality are not really related to the code from this pr.

**Comment by reduz** [comment]
Pr for other proposal welcome :P

**Comment by kubecz3k** [comment]
Yeah I'm working on it :)

**Pull request closed without merging**